### PR TITLE
Patch for mcsh5

### PR DESCRIPTION
**Pull request by yger**
I have to work with mcsh5 file, and sometimes the unit is not saved in the file. I don't know why, I did not generate the files myself, but I guess this can be quite common. Therefore, I think we should still be able to ready the files even without the proper unit

### CODE REVIEW

File: spikeinterface/extractors/mcsh5extractors.py [hunk 1]
```diff
@@ -124,7 +124,8 @@ def openMCSH5File(filename, stream_id):
     assert timestamps[0][0] < timestamps[0][2], 'Please check the validity of \'ChannelDataTimeStamps\' in the stream.'
     TimeVals = np.arange(timestamps[0][0], timestamps[0][2] + 1, 1) * Tick
 
-    assert Unit == b'V', 'Unexpected units found, expected volts, found {}'.format(Unit.decode('UTF-8'))
+    if not Unit == b'V':
```
Comment (alejoe91) [inline spikeinterface/extractors/mcsh5extractors.py:127]:
```suggestion
    if Unit != b'V':
```
?

File: spikeinterface/extractors/mcsh5extractors.py [hunk 2]
```diff
@@ -124,7 +124,8 @@ def openMCSH5File(filename, stream_id):
...
+        print('Unexpected units found, expected volts, found {}'.format(Unit.decode('UTF-8')))
```
Comment (alejoe91) [inline spikeinterface/extractors/mcsh5extractors.py:128]:
```suggestion
        print(f"Unexpected units found, expected volts, found {Unit.decode('UTF-8')}. Assuming Volts.")
```

**Comment by alejoe91** [comment]
Thanks @yger 

Just a couple of suggestions ;)

**Comment by yger** [comment]
perfect, fine for me

**Review by alejoe91 — APPROVED** [review]

**Comment by samuelgarcia** [comment]
I am not sure to like this.
What was the Unit in your case ? If it was uV or mV then we need to handle the gain.
It s important.

**Comment by alejoe91** [comment]
Right! Pierre can you print it and check?

**Pull request merged**